autoapi/mappers/python/parser.py

```diff
@@ -24,7 +24,7 @@ def _get_full_name(self, name):
     def _parse_file(self, file_path, condition):
         directory, filename = os.path.split(file_path)
         module_parts = []
-        if filename != "__init__.py":
+        if filename != "__init__.py" or filename != "__init__.pyi":
```

Comment:
This will always be true. This is equivalent to `if not (filename == "__init__.py" and filename == "__init__.pyi"):`, and a file cannot equal both names. Is this supposed to be an `and` rather than an `or`?

Reply:
Yes, you are right. I have added a commit for this problem